web/src/containers/Activities.js

```diff
@@ -29,7 +30,10 @@ const Activities = ({ activityIds, addActivity }) => (
         </button>
       </Collapsible>
       {activityIds.map((aId, idx) => (
-        <ActivityDetailAll key={aId} aId={aId} num={idx + 1} />
+        <Fragment key={aId}>
+          <ActivityDetailAll aId={aId} num={idx + 1} />
+          <ActivityDetailStandardsAndConditions aId={aId} />
```

Comment:
i think we want `StandardsAndConditions` nested within the `ActivityDetailAll` component, similar to these activity detail sections - https://github.com/18F/cms-hitech-apd/blob/master/web/src/containers/ActivityDetailAll.js#L19